src/sinks/console.rs

```diff
@@ -29,7 +29,7 @@ impl Default for Target {
 pub struct ConsoleSinkConfig {
     #[serde(default)]
     pub target: Target,
-    pub encoding: Option<Encoding>,
+    pub encoding: Encoding,
```

Comment:
@LucioFranco shouldn't this have been resolved in https://github.com/timberio/vector/pull/894? I assume we just missed this? Making sure this wasn't intentional.

Reply:
I believe I discussed this with you or luke but the reason this sink allows a null encoding is that it makes sense here to auto change how logs are printed since this is a debug sink. Happy to go the other way but for me this one actually made sense to dynamically choose since the downstream service is a human instead of a machine that might expect an encoding.

Reply:
I don't remember having that conversation. Do you have this documented anywhere? It should be in Github or Slack somewhere, correct?

Reply:
I'd have been ok with leaving it, but I only noticed this because the logic was complex enough that we skipped an entire case and weren't outputting anything at all. That convinced me we should simplify.